pkg/openshift/parser/template_test.go

```diff
@@ -113,22 +113,27 @@ var _ = Describe("template processing", func() {
 	})
 })
 
-const processedTmplWithDefaults = `kind: Template
+const processedOperatorTmplWithDefaults = `kind: Template
```

Comment:
Any reason to keep this vs reading the actual file?

Reply:
This one is processed - so have values replaced.